### DISCUSSION TITLE
Florisboard symbol layout

### DISCUSSION THREAD

**Discussion by mfathurr** [florisboard/florisboard#2575] [General]
Hi all,

First I would like to thank the creator of this app and all the contributors who have made this great app.

Next, I would like to ask whether the layout of the symbols in Florisboard can be customized to be like GBoard or manual customization according to the needs of each user?

![GridArt_20240919_211017224.jpg](https://github.com/user-attachments/assets/1ec9496a-f52d-46c3-b297-b861993c258a)

**Comment by patrickgold** [comment]
You can write your own JSON layouts and import them as a keyboard extension, but other than that not yet. In v0.5 addons support for keyboard extensions will be added alongside the new xml layouts, then this will be easier to customize.

**Reply by mfathurr** [reply]
Thanks for the answer. Ok. For ease of use in customizing layout of the symbols, I will wait until v0.5.

**Comment by savetier** [comment]
Any idea when 0.5 will be out?

**Reply by lm41** [reply]
When its ready. There are no ETAs.
